osm-oauth.server.js

```diff
@@ -29,13 +29,15 @@ OAuth.registerService('openstreetmap', 1, urls, function(oauthBinding) {
     accessToken: OAuth.sealSecret(oauthBinding.accessToken),
     accessTokenSecret: OAuth.sealSecret(oauthBinding.accessTokenSecret)
   };
+
   return {
     serviceData: serviceData,
     options: {
       profile: {
         username: identity.$.display_name,
+        picture: identity.img[0].$.href,
         description: identity.description[0],
-        languages: identity.languages,
+        languages: identity.languages[0].lang,
```

Comment:
a user can have multiple languages. Here you are selected the first one only.

Reply:
identity contains this
```javascript
{
	profile: {
		username: 'StefanoCudini',
		picture: 'https://www.openstreetmap.org/attachments/users/images/000/130/691/original/39c24d8cd7d1cb787df7f31ffd98c2c9.jpg',
		description: '',
		languages: ['it'],
		accountCreatedAt: '2009-06-01T18:03:32Z' 
	} 
}
```